api/nakadi-event-bus-api.yaml

```diff
@@ -1634,6 +1666,40 @@ definitions:
       purposes. We recommend logging this object in order to solve connection issues. Clients
       should not parse this structure.
 
+  CursorDistanceQuery:
+    required:
+      - partition
+      - initial_offset
+      - final_offset
+    properties:
+      partition:
+        type: string
+        description: |
+          Partition for which the offset distance is measured.
+      initial_offset:
+        type: string
+        description: |
+          Offset of the older event in the stream.
+      final_offset:
+        type: string
+        description: |
+          Offset of the newer event in the stream.
+
+  CursorDistanceResult:
```

Comment:
Does it make sense to extend `CursorDistanceQuery` and include some kind of additional field like `error` in case of error?